readme.md

```diff
@@ -136,7 +136,8 @@ Just do not define a read function.
 const multiplyCountAtom = atom(null, (get, set, by) => set(countAtom, get(countAtom) * by))
 
 function Controls() {
-  const [, multiply] = useAtom(multiplyCountAtom)
+  // causes unnecessary re-render: const [, multiply] = useAtom(multiplyCountAtom)
```

Comment:
No, it doesn't cause unnecessary re-renders.
We will keep using the original pattern for the basic usage.